user-guide/Advanced_Functionality/Swarming/SwarmingElements.md

```diff
@@ -4,17 +4,164 @@ uid: SwarmingElements
 
 # Swarming elements
 
-With DataMiner Swarming, you can swarm basic elements from one DataMiner Agent in a cluster to another. At present, the *Swarming* [soft-launch option](xref:SoftLaunchOptions) must be enabled for this.
+With DataMiner Swarming, you can swarm basic elements from one DataMiner Agent to another within a cluster. You can do so in DataMiner Cube or via an Automation script.
 
-When you are swarming an element so it gets hosted on a different DataMiner Agent, a temporary transition occurs.
-
-To indicate this, the following message will be displayed:
+When you are swarming an element so it gets hosted on a different DataMiner Agent, a temporary transition occurs. To indicate this, the following message will be displayed:
 
 ```txt
-'ElementName' is currently switching.
+'ElementName' is currently swarming.
            More info
 ```
 
-During this transition, the ability to open element cards or change element configuration will be temporarily suspended.
+During this transition, the ability to open element cards or change element configuration for the involved element will be temporarily suspended. Once the element migration is complete, it will become accessible again.
+
+> [!NOTE]
+>
+> - To be able to trigger swarming for an element, you need the *Swarming* user permission as well as config rights on the element.
+> - Swarming is currently only possible with regular elements. See [Limitations](xref:Swarming#limitations).
+
+## Swarming elements in DataMiner Cube
+
+To swarm elements in DataMiner Cube:
+
+1. Go to *System Center* > *Agents* > *Status* and click the *Swarm* button in the lower right corner.
```

Comment:
Does this replace the element migration functionality as described here: [https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Systems/Migrating_elements_in_a_DataMiner_System.html](https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Systems/Migrating_elements_in_a_DataMiner_System.html)? RN 38467 seems to suggest as much

Reply:
I've now added this info. However, one thing that still isn't clear and that users might wonder about is what advantages there might be to using swarming instead of migrating using DELT. Why should a user choose the one over the other?

Reply:
Migrating using DELT is the combination of a few steps:
- the element gets exported into a dmimport package (containing the element config)
- the element gets deleted on the original agent
- the DELT package gets copied over to the new agent
- the DELT dmimport package is imported on the new agent

Migrating using swarming:
- Element configuration remains in the shared database
- First agent unloads the element
- New agent loads the element

Migration using DELT is quite error-prone. For example, if the export and delete works but the copy/import does not, you would end up with a missing element. There's a lot less risk in the swarm action as it doesn't delete any configuration.

Does this help?

Reply:
I've added a note with this info on [the migration page](https://github.com/SkylineCommunications/dataminer-docs/pull/3964/files#diff-fed342fec7ac30732731d004e4a91a6a89f06fede535eb96939e94e6d7e355b4) now. Does this look OK for you?